src/components/filter/FilterForm.tsx

```diff
@@ -14,6 +14,15 @@ import { ExplicitNamingFilterForm } from './explicitNaming/ExplicitNamingFilterF
 import { ExpertFilterForm } from './expert/ExpertFilterForm';
 import { FilterType } from './constants/FilterConstants';
 import { unscrollableDialogStyles } from '../dialogs';
+import yup from '../../utils/yupConfig';
+import { MAX_CHAR_DESCRIPTION } from '../../utils/constants/UIconstants';
+
+export const FilterSchema = {
```

Comment:
```suggestion
export const HeaderFilterSchema = {
```

and move to HeaderFilterForm